### ISSUE TITLE
Gantt Diagram incorrect position of milestones

### ISSUE THREAD

**Issue by glodov** [plantuml/plantuml#321]
Hello!

The position of Milestones are incorrect in the `3. Implementation & Coding` Phase.
They should be rendered right below the end of the `Developing all the tasks`.
```
[Development complete] happens at [Coding]'s end and is colored in Black
[3rd payment] happens at [Coding]'s end and is colored in Black
```

The source code of the Template.puml you can find here: 
 https://github.com/yaroinfo/guidelines/blob/master/management/Project-Management-Gantt.puml

You are welcome to use this template for your management with your team and customers, but Star the Yaro's repository and refer to it on your website in partners section :)

**Comment by Phrogz** [comment]
Here's a pared down test case:

```
@startuml
Project starts the 20th of May 2020

[Prototype] starts at 2020-6-6 and ends 2020-6-11
[Developing] lasts 5 days
[Complete] happens at [Developing]'s end
[Define] lasts 1 day

[Define] starts at [Prototype]'s end
[Developing] starts at [Define]'s end
@enduml
```

produces this result:
![image](https://user-images.githubusercontent.com/14778/98746195-3bb69f00-2372-11eb-9793-b563d4e66ba8.png)

Moving the line `[Complete] happens at [Developing]'s end` to be the last line of this test case fixes the date problem, causing the milestone to correctly be placed at the end of `Developing`, but of course on a different line than desired.

A workaround is to have a line declaring `Complete` on the line you want, but later setting its constraint, like so:

```
@startuml
Project starts the 20th of May 2020

[Prototype] starts at 2020-6-6 and ends 2020-6-11
[Developing] lasts 5 days
[Complete] lasts 1 day
[Define] lasts 1 day

[Define] starts at [Prototype]'s end
[Developing] starts at [Define]'s end
[Complete] happens at [Developing]'s end
@enduml
```

![image](https://user-images.githubusercontent.com/14778/98746355-8b956600-2372-11eb-95dc-d55ac77d99a3.png)

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now answered.
- Thanks @glodov for the report.
- Thanks @Phrogz  for your answer.
- Thanks PlantUML team for your work.

Regards.

**Comment by Phrogz** [comment]
I don’t think this should be closed. I provided a workaround, but I also provided a simple repro case showing the issue being described.
